### PR TITLE
Update manage-connections-from-windows-operating-system-components-to…

### PR DESCRIPTION
**Pull request by TomLayson**
…-microsoft-services.md

Minor text change

### CODE REVIEW

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 1]
```diff
@@ -1266,7 +1266,7 @@ In the **Feedback & Diagnostics** area, you can choose how often you're asked fo
 To change how frequently **Windows should ask for my feedback**:
 
 > [!NOTE]
-> Feedback frequency only applies to user-generated feedback, not diagnostic and usage data sent from the device.
+> Feedback frequency only applies to user-generated feedback, not diagnostic and usage data sent from the device.
```
Comment (illfated) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:1269]: This extra space is not required and can be safely omitted.
```suggestion
> Feedback frequency only applies to user-generated feedback, not diagnostic and usage data sent from the device.
```

**Comment by illfated** [comment]
Looks fairly good, just a couple of questions of minor importance regarding the added table:
- Would it be OK to align the cell dividers between column 1 and column 2, for easier future editing?
- Would it be desirable to keep the text within one and the same row, while still having a Line Break/NewLine between the current "row 1" and "row 2" of the values part of the table?

Example to accompany question 2:

| Policy                        | Group Policy Path  | Registry Path                               |
|-------------------------------|--------------------|---------------------------------------------|
| **SearchSuggestEnabled**      |  Computer Configuration/Administrative Templates/Windows Component/Microsoft Edge - Enable search suggestions<br> **Set to Disabled**  | HKEY_LOCAL_MACHINE \SOFTWARE\Policies\Microsoft\Edge<br>**REG_DWORD name: SearchSuggestEnabled Set to 0** |
| **AutofillAddressEnabled**    |  Computer Configurations/Administrative Templates/Windows Component/Microsoft Edge - Enable AutoFill for addresses<br> **Set to Disabled** | HKEY_LOCAL_MACHINE \SOFTWARE\Policies\Microsoft\Edge<br>  **REG_DWORD name: AutofillAddressEnabled Set to 0** |
| **AutofillCreditCardEnabled** |  Computer Configurations/Administrative Templates/Windows Component/Microsoft Edge - Enable AutoFill for credit cards<br> **Set to Disabled** | HKEY_LOCAL_MACHINE \SOFTWARE\Policies\Microsoft\Edge<br> **REG_DWORD name: AutofillCreditCardEnabled Set to 0** |

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 2]
```diff
@@ -592,6 +592,48 @@ Alternatively, you can configure the following Registry keys as described:
 
 For a complete list of the Microsoft Edge policies, see [Available policies for Microsoft Edge](/microsoft-edge/deploy/available-policies).
 
+### <a href="" id="bkmk-edgegp"></a>13.2 Microsoft Edge Enterprise
+
+For a complete list of the Microsoft Edge policies, see [Microsoft Edge and privacy: FAQ](https://docs.microsoft.com/en-us/microsoft-edge/deploy/available-policies).
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:597]:
```suggestion
For a complete list of the Microsoft Edge policies, see [Group Policy and Mobile Device Management (MDM) settings for Microsoft Edge](/microsoft-edge/deploy/available-policies). 
```

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 3]
```diff
@@ -592,6 +592,48 @@ Alternatively, you can configure the following Registry keys as described:
 
 For a complete list of the Microsoft Edge policies, see [Available policies for Microsoft Edge](/microsoft-edge/deploy/available-policies).
 
+### <a href="" id="bkmk-edgegp"></a>13.2 Microsoft Edge Enterprise
+
+For a complete list of the Microsoft Edge policies, see [Microsoft Edge and privacy: FAQ](https://docs.microsoft.com/en-us/microsoft-edge/deploy/available-policies). 
+
+> [!Important]
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:599]:
```suggestion
> [!IMPORTANT]
```

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 4]
```diff
@@ -592,6 +592,48 @@ Alternatively, you can configure the following Registry keys as described:
...
+> - The following settings are applicable to Microsoft Edge version 77 or later.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:600]:
```suggestion
> - The following settings are applicable to Microsoft Edge version 77 or later. 
```
```suggestion
> - The following settings are applicable to Microsoft Edge version 77 or later.
```

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 5]
```diff
@@ -592,6 +592,48 @@ Alternatively, you can configure the following Registry keys as described:
 
 For a complete list of the Microsoft Edge policies, see [Available policies for Microsoft Edge](/microsoft-edge/deploy/available-policies).
 
+### <a href="" id="bkmk-edgegp"></a>13.2 Microsoft Edge Enterprise
+
+For a complete list of the Microsoft Edge policies, see [Microsoft Edge and privacy: FAQ](https://docs.microsoft.com/en-us/microsoft-edge/deploy/available-policies). 
+
+> [!Important]
+> - The following settings are applicable to Microsoft Edge version 77 or later. 
+> - For details on supported Operating Systems see Microsoft Edge supported Operating Systems
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:601]:
```suggestion
> - For details on supported Operating Systems, see [Microsoft Edge supported Operating Systems](/deployedge/microsoft-edge-supported-operating-systems).
```

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 6]
```diff
@@ -592,6 +592,48 @@ Alternatively, you can configure the following Registry keys as described:
...
 For a complete list of the Microsoft Edge policies, see [Available policies for Microsoft Edge](/microsoft-edge/deploy/available-policies).
 
+### <a href="" id="bkmk-edgegp"></a>13.2 Microsoft Edge Enterprise
+
+For a complete list of the Microsoft Edge policies, see [Microsoft Edge and privacy: FAQ](https://docs.microsoft.com/en-us/microsoft-edge/deploy/available-policies). 
+
+> [!Important]
+> - The following settings are applicable to Microsoft Edge version 77 or later. 
+> - For details on supported Operating Systems see Microsoft Edge supported Operating Systems 
+> - These policies require the Microsoft Edge administrative templates to be applied. For more information on administrative templates for Microsoft Edge see Configure Microsoft Edge policy settings on Windows
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:602]:
```suggestion
> - These policies require the Microsoft Edge administrative templates to be applied. For more information on administrative templates for Microsoft Edge, see [Configure Microsoft Edge policy settings on Windows](/deployedge/configure-microsoft-edge).
```

File: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md [hunk 7]
```diff
@@ -592,6 +592,48 @@ Alternatively, you can configure the following Registry keys as described:
...
 
+### <a href="" id="bkmk-edgegp"></a>13.2 Microsoft Edge Enterprise
+
+For a complete list of the Microsoft Edge policies, see [Microsoft Edge and privacy: FAQ](https://docs.microsoft.com/en-us/microsoft-edge/deploy/available-policies). 
+
+> [!Important]
+> - The following settings are applicable to Microsoft Edge version 77 or later. 
+> - For details on supported Operating Systems see Microsoft Edge supported Operating Systems 
+> - These policies require the Microsoft Edge administrative templates to be applied. For more information on administrative templates for Microsoft Edge see Configure Microsoft Edge policy settings on Windows 
+> - Devices must be domain joined for some of the policies to take effect.
```
Comment (JohanFreelancer9) [inline windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md:603]:
```suggestion
> - Devices must be domain joined for some of the policies to take effect.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by siosulli** [comment]
Thanks @JohanFreelancer9 I have committed your suggestions and merged the PR.

**Pull request merged**